addons/web/static/src/js/views/kanban/kanban_quick_create.js

```diff
@@ -28,13 +28,18 @@ var AbstractQuickCreate = Widget.extend({
      *
      * @private
      */
-    _add: function () {
+     _get_data: function(value) {
```

Comment:
rename _get_data into _getData, there should be a space between function and (, the value parameter should be documented, and this function should be placed after the _add function